pom.xml

```diff
@@ -90,4 +89,18 @@
             </plugins>
         </pluginManagement>
     </build>
+    <profiles>
+        <profile>
+            <id>it</id>
+            <activation>
+                <property>
+                    <name>performRelease</name>
```

Comment:
this property is already set I presume

Reply:
Yeah, it has been set in `release.ym` when running 
```
mvn -B release:perform -Darguments=-DperformRelease -DperformRelease -Prelease
```